src/histolab/tiler.py

```diff
@@ -67,6 +68,32 @@ def box_mask(self, slide: Slide) -> np.ndarray:
     def extract(self, slide: Slide):
         raise NotImplementedError
 
+    def locate_tiles(self, slide: Slide) -> Image:
+        """Place tiles references on the slide thumbnail image
+
+        Parameters
+        ----------
+        slide : Slide
+            Slide reference where placing the tiles
+
+        Returns
+        -------
+        img
+            PIL Image with tiles annotations
+        """
+        if not os.path.exists(slide.thumbnail_path):
+            slide.save_thumbnail()
+        tiles_coords = (tc[1] for tc in self._tiles_generator(slide))
+        img = Image.open(slide.thumbnail_path)
+        img.putalpha(128)
```

Comment:
Do you think alpha should be a parameter?

Reply:
Considering the wsi(s) color heterogeneity maybe the alpha as param can be really useful !